auth0_flutter/example/android/app/src/androidTest/kotlin/com/auth0/auth0_flutter_example/SmokeTest.kt

```diff
@@ -0,0 +1,90 @@
+package com.auth0.auth0_flutter_example
+
+import android.app.UiAutomation
+import android.content.Context
+import android.content.Intent
+import android.os.Environment
+import android.os.Environment.DIRECTORY_PICTURES
+import android.widget.Button
```

Comment:
```suggestion
import android.content.Intent
import android.widget.Button
```
These are not needed anymore.